packages/pluginsdk-peerdeps/USAGE.txt

```diff
@@ -0,0 +1,29 @@
+yarn usage
```

Comment:
Make sure to sync this with the README (either both `yarn help` or both `yarn usage`